bedrock/base/templates/includes/global-nav.html

```diff
@@ -248,14 +248,22 @@ <h3 class="summary" data-id="technology">
                   </a>
                 </li>
                 <li>
-                  <a href="{{ url('mozorg.developer') }}" data-link-type="nav" data-link-name="Mozilla Developers" data-link-group="Technology">
-                    {{ _('Mozilla Developers') }}
-                  </a>
+                    <a href="{{ url('mozorg.developer') }}" data-link-name="Developer Hub" data-link-type="nav" data-link-position="subnav">{{ _('Developer Hub') }}</a>
                 </li>
                 <li>
-                  <a href="https://developer.mozilla.org/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=global-nav&amp;utm_content=MDN" data-link-type="nav" data-link-name="MDN" data-link-group="Technology">
-                    <abbr title="{{ _('Mozilla Developer Network') }}">{{ _('MDN') }}</abbr>
-                  </a>
+                  <a href="https://research.mozilla.org/" data-link-name="Research" data-link-type="nav" data-link-position="subnav">{{ _('Research') }}</a>
+                </li>
+                <li>
+                  <a href="https://vr.mozilla.org/" data-link-name="Virtual Reality" data-link-type="nav" data-link-position="subnav">{{ _('Virtual Reality') }}</a>
```

Comment:
Just notices this does not need the `data-location` but does need the `data-group` attribute